### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by sakshamtaneja21**
| Q             | A
| ------------- | ---
| Bug fix?      | yes/no 
| New feature?  | yes/no 
| BC breaks?    | no
| Deprecations? | yes/no
| Tickets       | fixes #... 
| License       | MIT
| Doc PR        | api-platform/docs#...

### CODE REVIEW

File: distribution/index.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ This starts the following services:
 
 | Name     | Description                                                       | Port(s)                                                     | Environment(s)                                     |
 |----------|-------------------------------------------------------------------|-------------------------------------------------------------|----------------------------------------------------|
-| php      | The API with PHP, PHP-FPM 7.3, Composer and sensitive configs     | n/a                                                         | all                                                |
+| PHP      | The API with PHP, PHP-FPM 7.3, Composer and sensitive configs     | n/a                                                         | all                                                |
```
Comment (antograssiot) [inline distribution/index.md]: this should be reverted I think, that the name of the service in the docker-compose

Comment (alanpoulain) [inline distribution/index.md:78]:
```suggestion
| php      | The API with PHP, PHP-FPM 7.3, Composer and sensitive configs     | n/a                                                         | all                                                |
```

File: distribution/index.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ API Platform uses these model classes to expose and document a web API having a
 One more thing, before we start: as the API Platform distribution includes [the Symfony framework](https://symfony.com),
 it is compatible with most [Symfony bundles](https://flex.symfony.com)
 (plugins) and benefits from [the numerous extensions points](../core/extending.md) provided by this rock-solid foundation (events, DIC...).
-Adding features like custom, service-oriented, API endpoints, JWT or OAuth authentication, HTTP caching, mail sending or
+Adding features like a custom, service-oriented, API endpoints, JWT or OAuth authentication, HTTP caching, mail sending or
```
Comment (alanpoulain) [inline distribution/index.md:50]:
```suggestion
Adding features like custom or service-oriented API endpoints, JWT or OAuth authentication, HTTP caching, mail sending or
```

**Comment by alanpoulain** [comment]
Thanks.

**Pull request merged**